slides/r-adoption-jan2023.qmd

```diff
@@ -0,0 +1,229 @@
+  data-background-opacity: "0.5"
+  data-background-position: 80% 10%
+editor: visual
+---
+
+## Agenda
+
+-   Introduction and Overview of AWA SWE
+
+-   Mix Models for Repeated Measures - Why is it a Problem?
+
+-   MMRM Package
+
+    -   Why this is not "yet another package"
+
+    -   Long Term Perspective
+
+    -   Comparing MMRM R Package to SAS - Demo
+
+-   Closing and Next Steps
+
+## Software Engineering in Biostatistics
+
+-   Open-source software has gained increasing popularity in Biostatistics over the last two decades
+    -   Pros: rapid uptake of novel statistical methods and unprecedented opportunities for collaboration and innovation
+    -   Cons: users face huge variability in software quality, in particular reliability, efficiency and maintainability
+-   Developing high-quality software with good coding practices, reproducible outputs, and self-sufficient documentation is critical to inform clinical and regulatory decisions
+
+## How to deal with these issues?
+
+-   To deal with the issues of statistical quality assurances for R packages and creating high quality statistical software a group came together to create through the ASA called the "Software Engineering Working Group" SWE WG to look R packages from a statistical point of view
+
+## ASA BIOP Software Engineering working group (SWE WG)
+
+-   An official working group of the ASA Biopharmaceutical Section
+-   Formed in August 2022
+-   Cross-industry collaboration with more than 30 members from over 20 organizations
+-   Home page at [rconsortium.github.io/asa-biop-swe-wg](https://rconsortium.github.io/asa-biop-swe-wg)
+
+## Goals
+
+-   **Primary Goal**: Collaborate to engineer R packages that implement important statistical methods to fill in critical gaps
+
+-   **Secondary Goal**: Develop and disseminate best practices for engineering high-quality open-source statistical software
+
+## SWE WG Activities
+
+-   First R package `mmrm` was published on CRAN in October 2022 and updated in December
+    -   We aim to estabilish this package as a new standard for fitting mixed models for repeated measures (MMRM)
+    -   We have been developing and adopting best practices for software in the `mmrm` package, and open sourced it at [github.com/openpharma/mmrm](https://github.com/openpharma/mmrm)
+    -   Currently under active development to add more features
+
+## WHY MMRM?
+
+-   Mixed Models for Repeated Measures (MMRM) is a popular choice for analyzing longitudinal continuous outcomes in randomized clinical trials
+-   No great R Package - initially though that the MMRM problem was solved by using a combination of `lme4` and `lmerTest`
+-   Learned that this approach failed on Ophthalamology data (slow, did not converge
+-   `nlme` does not give Satterthwaite adjusted degrees of freedom, has convergence issues, and with `emmeans` it is only approximate
+-   tried to extend `glmmTMB` to calculate Satterthwaite adjusted degrees of freedom
+
+## Idea
+
```

Comment:
I would suggest before this slide show the one from R Govys: https://github.com/RConsortium/asa-biop-swe-wg/blob/main/slides/r-govys-jan2023.qmd#L90

Reply:
incorporated feedback